arviz/stats/stats.py

```diff
@@ -305,27 +307,26 @@ def _ic_matrix(ics, ic_i):
     return rows, cols, ic_i_val
 
 
-def hpd(ary, credible_interval=0.94, circular=False):
+def hpd(ary, credible_interval=0.94, multimodal=True):
```

Comment:
I think it is still useful to have circular karg, it is different than multimodal right?

Reply:
The circular correction will work with a unimodal distribution, but not necessary with a multimodal one. The circular correction shifts the distribution by its mean in an attempt to move the interval away from the "borders" . Anyway I can add it again with a warning, or think of a less trickier way to compute it.